packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionInvoke.test.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { ForceFunctionInvoke as Unit } from '../../../../src/commands/functions/forceFunctionInvoke';
```

Comment:
I don't see we use this pattern elsewhere. I don't have a very strong preference against this pattern, but I feel if we just use the original names the code readability would still be on par or slightly better

Reply:
I thought this actually improves readability by clearly showing what you are testing. I can revert it if not everyone is onboard and is something we will follow going forward.

Reply:
When someone reads the code, a full name gives the person more context at a place where it's tested. Whereas if using a generic unit name, you'll have to refer to somewhere else. If you look at a single line where `Unit` was involved, is it easier to tell what the statement is doing than using a full name?

In addition, we are in an integration test suite here although we are using it for unit tests.